package.json

```diff
@@ -53,30 +53,29 @@
     "api:review": "sh ./scripts/api.sh -p 9443 -t review",
     "api:stage": "sh ./scripts/api.sh -p 5001 -t stage",
     "api:update": "npm run api:build; sh ./scripts/api.sh -t update",
-    "build": "run-s -l build:clean build:version build:css build:js build:finish test:integration",
+    "build": "run-s -l build:clean build:docs test:docs build:version build:css build:js build:finish test:integration",
     "build:clean": "rm -rf -- \"$(pwd)\"/build",
     "build:css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/styles/index.scss -o src/styles/.css",
+    "build:docs": "node ./scripts/quipudocs.js",
```

Comment:
```
cd ../quipucords-ui;yarn;yarn build
yarn install v1.16.0
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
warning Resolution field "jest@23.6.0" is incompatible with requested version "jest@20.0.4"
success Already up-to-date.
✨  Done in 0.71s.
yarn run v1.16.0
$ run-s -l build:clean build:docs test:docs build:version build:css build:js build:finish test:integration
[build:clean     ] $ rm -rf -- "$(pwd)"/build
[build:docs      ] $ node ./scripts/quipudocs.js
[build:docs      ] internal/modules/cjs/loader.js:626
[build:docs      ]     throw err;
[build:docs      ]     ^
[build:docs      ]
[build:docs      ] Error: Cannot find module 'quipudocs'
[build:docs      ] Require stack:
[build:docs      ] - /Users/cmyers/Desktop/projects/quipucords-ui/scripts/quipudocs.js
[build:docs      ]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
[build:docs      ]     at Function.Module._load (internal/modules/cjs/loader.js:527:27)
[build:docs      ]     at Module.require (internal/modules/cjs/loader.js:681:19)
[build:docs      ]     at require (internal/modules/cjs/helpers.js:16:16)
[build:docs      ]     at Object.<anonymous> (/Users/cmyers/Desktop/projects/quipucords-ui/scripts/quipudocs.js:2:40)
[build:docs      ]     at Module._compile (internal/modules/cjs/loader.js:774:30)
[build:docs      ]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
[build:docs      ]     at Module.load (internal/modules/cjs/loader.js:641:32)
[build:docs      ]     at Function.Module._load (internal/modules/cjs/loader.js:556:12)
[build:docs      ]     at Function.Module.runMain (internal/modules/cjs/loader.js:837:10) {
[build:docs      ]   code: 'MODULE_NOT_FOUND',
[build:docs      ]   requireStack: [
[build:docs      ]     '/Users/cmyers/Desktop/projects/quipucords-ui/scripts/quipudocs.js'
[build:docs      ]   ]
[build:docs      ] }
[build:docs      ] error Command failed with exit code 1.
[build:docs      ] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
After talking with dave it looks like something didn't get merged in quipudocs.

Reply:
Yup, so after some more digging. Looks like we were referencing an "older" version of quipudocs via git commit hash. Since we really only pull the "master" branch with this PR, we'll need a subsequent PR to flip over to Quipudocs tags, making debugging a little bit easier. The reference for master has now been updated. 

NOTE:  need follow up PR to incorporate the latest Quipudocs changes through a more recent tag.